dropwizard-hibernate/src/main/java/io/dropwizard/hibernate/UnitOfWorkContext.java

```diff
@@ -0,0 +1,24 @@
+package io.dropwizard.hibernate;
+
+import org.hibernate.SessionFactory;
+
+import static java.util.Objects.requireNonNull;
+
+public class UnitOfWorkContext {
+    private final UnitOfWork unitOfWork;
+    private final SessionFactory sessionFactory;
+
+    UnitOfWorkContext(UnitOfWork unitOfWork,
+                      SessionFactory sessionFactory) {
+        this.unitOfWork = requireNonNull(unitOfWork);
+        this.sessionFactory = requireNonNull(sessionFactory);
+    }
+
+    public UnitOfWork getUnitOfWork() {
+        return unitOfWork;
+    }
+
+    public SessionFactory getSessionFactory() {
+        return sessionFactory;
+    }
```

Comment:
I don't think that equals/hashcode/toString make sense here but correct me if I'm wrong.